rust/ql/integration-tests/query-suite/rust-security-extended.qls.expected

```diff
@@ -0,0 +1,27 @@
+ql/rust/ql/src/queries/diagnostics/AstConsistencyCounts.ql
+ql/rust/ql/src/queries/diagnostics/CfgConsistencyCounts.ql
+ql/rust/ql/src/queries/diagnostics/DataFlowConsistencyCounts.ql
+ql/rust/ql/src/queries/diagnostics/ExtractedFiles.ql
+ql/rust/ql/src/queries/diagnostics/ExtractionErrors.ql
+ql/rust/ql/src/queries/diagnostics/ExtractionWarnings.ql
+ql/rust/ql/src/queries/diagnostics/SsaConsistencyCounts.ql
+ql/rust/ql/src/queries/diagnostics/UnextractedElements.ql
+ql/rust/ql/src/queries/diagnostics/UnresolvedMacroCalls.ql
+ql/rust/ql/src/queries/security/CWE-020/RegexInjection.ql
+ql/rust/ql/src/queries/security/CWE-022/TaintedPath.ql
+ql/rust/ql/src/queries/security/CWE-089/SqlInjection.ql
+ql/rust/ql/src/queries/security/CWE-311/CleartextTransmission.ql
+ql/rust/ql/src/queries/security/CWE-312/CleartextLogging.ql
+ql/rust/ql/src/queries/security/CWE-327/BrokenCryptoAlgorithm.ql
+ql/rust/ql/src/queries/security/CWE-328/WeakSensitiveDataHashing.ql
+ql/rust/ql/src/queries/security/CWE-770/UncontrolledAllocationSize.ql
+ql/rust/ql/src/queries/security/CWE-825/AccessInvalidPointer.ql
+ql/rust/ql/src/queries/summary/LinesOfCode.ql
+ql/rust/ql/src/queries/summary/LinesOfUserCode.ql
+ql/rust/ql/src/queries/summary/NumberOfFilesExtractedWithErrors.ql
+ql/rust/ql/src/queries/summary/NumberOfSuccessfullyExtractedFiles.ql
+ql/rust/ql/src/queries/summary/QuerySinkCounts.ql
+ql/rust/ql/src/queries/summary/SummaryStats.ql
+ql/rust/ql/src/queries/summary/SummaryStatsReduced.ql
+ql/rust/ql/src/queries/telemetry/DatabaseQualityDiagnostics.ql
+ql/rust/ql/src/queries/telemetry/ExtractorInformation.ql
```

Comment:
These `.expected` files will need updating fairly frequently, e.g. every time we add, move or sufficiently change the metadata for a query.  Is the intent of this to make these changes more visible?  To who?